csrankings-y.csv

```diff
@@ -613,6 +613,7 @@ Yu Li 0006,Chinese University of Hong Kong,https://liyu95.com,8YHZx-AAAAAJ
 Yu Lin 0001,Australian National University,https://cecs.anu.edu.au/people/yu-lin,KDwSQ0gAAAAJ
 Yu Long 0001,Shanghai Jiao Tong University,https://www.facebook.com/public/Long-Yu/school/Shanghai-Jiao-Tong-University-109235979095677,p1rD-B0AAAAJ
 Yu Mong,City University of Hong Kong,https://www.cs.cityu.edu.hk/profile/csymong.html,NOSCHOLARPAGE
+Yu Pei,The Hong Kong Polytechnic University,http://www4.comp.polyu.edu.hk/~csypei/index.html,rBGz-XoAAAAJ

```

Comment:
Needs a disambiguation suffix AND you need an extra entry for the DBLP alias. See https://dblp.org/pid/76/8304-1.html